src/components/SingleBlog/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from "react";
+import "./index.scss";
+import { Container, Col, Row } from "react-bootstrap";
+import { useParams } from "react-router-dom";
+import { ReactComponent as FacebookIcon } from "../../images/icons/facebookIcon.svg";
+import { ReactComponent as TwitterIcon } from "../../images/icons/twitterIcon.svg";
+import { ReactComponent as LinkedInIcon } from "../../images/icons/linkedInIcon.svg";
+import { ReactComponent as InstagramIcon } from "../../images/icons/instagramIcon.svg";
+
+const RELATED_POSTS_TO_SHOW = 4;
+
+export default function SingleBlog(props) {
+  const params = useParams();
+  const [blogPost, setBlogPosts] = useState();
+  const allBlogPosts = async () => {
```

Comment:
is it single blog or all blog posts? it's confusing

Reply:
it's a single blog that we have to fetch through posts to have it. I can't see where the confusing part is, maybe the s in setBlogPosts, I can remove it if it will make sense to you

Reply:
no, what confuses me is that are calling this function allBlogPosts while it fetches one blog post based on params.id. better name would be fetchSingleBlogPost or smth like this